pkgs/tools/security/pcsclite/default.nix

```diff
@@ -10,6 +10,7 @@
 , polkit
 , systemdMinimal
 , IOKit
+, polkitSupport ? true
```

Comment:
```suggestion
, polkitSupport ? stdenv.isLinux
```

Reply:
Actually polkit and dbus should be portable: the package meta says `platforms.unix`.

Reply:
> Actually polkit and dbus should be portable: the package meta says `platforms.unix`.

Do people really use dbus on Darwin?

(sorry if this is a stupid question; I got rid of my last Mac ~5 years ago)

Reply:
> Do people really use dbus on Darwin?

No idea, but if it (supposedly) works why not? Also, there's BSD too.

Reply:
Right now this enables polkit unconditional on darwin which it didn't before and which currently does not work. I would suggest to accept my change so that someone else can worry about polkit support on darwin and the build does not get broken.

Reply:
On second thought, it will probably have the same issue as on linux if the daemon not running. Maybe polkit should be disabled by default.